library/core/src/ffi/va_list.rs

```diff
@@ -94,97 +113,30 @@ crate::cfg_select! {
     // - apple aarch64 (see https://github.com/rust-lang/rust/pull/56599)
     // - windows
     // - uefi
-    // - any other target for which we don't specify the `VaListImpl` above
+    // - any other target for which we don't specify the `VaList` above
     //
     // In this implementation the `va_list` type is just an alias for an opaque pointer.
     // That pointer is probably just the next variadic argument on the caller's stack.
     _ => {
         /// Basic implementation of a `va_list`.
```

Comment:
I don't think this is urgent, but the comments on the `VaList` definitions should probably be consistent and not show implementation details? We can put a doc comment in an external file and `include_str!` it to prevent duplication.